browser/resources/resource_ids

```diff
@@ -25,4 +25,8 @@
   "brave/components/brave_rewards/resources.grd": {
     "includes": [38000],
   },
+  "brave/components/brave_sync/resources.grd": {
+    "includes": [37000],
+  }
+  # brave sync 37000
```

Comment:
why is there a comment for brave sync? We only use the comments for grd files with `resources_ids = ""` which we should avoid using in general